### PR TITLE
feat(renovate): add extension for group:allNonMajor

### PR DESCRIPTION
**Pull request by clok**
This change enables the [`group:allNonMajor`](https://docs.renovatebot.com/presets-group/#groupallnonmajor) feature within renovate. This will roll up all non-major changes into a single PR, rather than N PRs, one for each patch.

I like this feature as it will reduce noise from renovate and provide a convenient way group many changes in one.

Examples: 
Go project: https://github.com/GoodwayGroup/gw-aws-audit/pull/40
Node project: https://github.com/GoodwayGroup/lib-hapi-rollbar/pull/65

### CODE REVIEW

**Review by jpiepkow — APPROVED** [review]

**Review by paulojblack — APPROVED** [review]
This looks great! A thought and a question:

1) Should we establish a rule of thumb for when an update should be tested locally before merging the PR? Do we want to do this for all rolled up updates or do it on a "go with your gut" basis?
2) Say out of 10 rolled up updates, one updated lib causes problems which are non-trivial to sort out. I see it as fine to delay the entire bundle of updates until we have time to understand the breaking lib. However, is there a relatively easy way to remove an update from the bundle while keeping the rest?

**Comment by clok** [comment]
> 1. Should we establish a rule of thumb for when an update should be tested locally before merging the PR? Do we want to do this for all rolled up updates or do it on a "go with your gut" basis?

I leave that to teams to decide as they manage repos. "Go with your gut" works for me for 2 reasons. 1) We have tests 2) We have staging. A minor patch issue should be caught by those. If not, the team should look at the tests and staging to understand why.

> 2. Say out of 10 rolled up updates, one updated lib causes problems which are non-trivial to sort out. I see it as fine to delay the entire bundle of updates until we have time to understand the breaking lib. However, is there a relatively easy way to remove an update from the bundle while keeping the rest?

Initial feeling is that you could manually create a PR with the ones that work and move that through the system. If a single patch is known to cause an issue and it will take time to work it out then I don't believe it should block the others. Renovate is good at updating PRs once a change to the mainline occurs.

**Review by bnormington — APPROVED** [review]
👍  I'm looking forward to the time saving this will give us

**Pull request merged**
